<launch>
  <arg name="ns" default="audio"/>
  <node name="audio_player" pkg="audio_play" type="raw_audio_player"
	clear_params="true" output="screen">
    <param name="audio_in_rate" value="48000"/>
    <param name="audio_in_format" value="S16LE"/>
    <param name="audio_in_depth" value="16"/>
    <param name="num_audio_in_channels" value="1"/>
    <rosparam param="out_channels"> [0]
    </rosparam>
    <remap from="~audio_stamped"  to="/audio/audio_stamped"/>
  </node>
</launch>
